e2e-tests/config/config.json

```diff
@@ -34,6 +34,15 @@
         "publicKey": "GD3G7WKBQOQGKXWNVGAAQHJPSQEDYOAN7VP2IZTC6CKKUMQ62L4E5WLY",
         "secretKey": "SD3M4GVODVCEBOSCDTP2RCT7YXQJPH35CDSEBQHMCZXUZCALKMKG5MOT"
       }
+    },
+    {
+      "protocol": "evm",
```

Comment:
`"ethereum"`, but along with https://github.com/calimero-network/core/pull/1142#discussion_r1995920234, can be renamed in a follow-up PR